src/patterns/question-pages/index.md.njk

```diff
@@ -83,6 +83,19 @@ For example, ‘About you’
 
 You can also learn more about how starting with [one thing per page](https://www.gov.uk/service-manual/design/form-structure#start-with-one-thing-per-page) helps users in the GOV.UK Service Manual.
 
+#### Asking complex questions without using hint text
+
+Do not use hint text if you need to give a lengthy explanation with lists and paragraphs. Screen readers read out the entire text when users interact with the form element. This could frustrate users if the text is long.
+
+If you're asking a question that needs a detailed explanation, use:
+* a `h1` heading that's a statement (for example, 'Add undergraduate degree') rather than a question, and (where possible) is consistent with other `h1` headings in your service
```

Comment:
My bad - this first bullet is long-winded.

@christopherthomasdesign The bit about remaining consistent with other services feels like an optional suggestion, whereas the bit about using a statement feels more like a strong recommendation.

I wonder could we rejig the bullet to something like:

```suggestion
* a `h1` heading that's a statement (for example, 'Add undergraduate degree') rather than a question
```

...and then bump the 'consistency' content to a standalone sentence under the last of the bullets, and above the example. Like so:

> Where possible, you should also make sure your `h1` heading is consistent with other `h1` headings in your service.